documents/2-RTB/Documentazione_Esterna/Glossario.typ

```diff
@@ -203,6 +290,13 @@ uniformità nell'approccio e nel risultato finale.
 #pagebreak()
 
 = #LTG("O")
+
+== #LTG("Onboarding")
+
+Processo per integrare nuovi membri in un team o progetto, fornendo loro le informazioni e le risorse necessarie per iniziare.
```

Comment:
"per iniziare a lavorare in modo efficace." aggiungerei alla fine. riga 296

Reply:
done